### PR TITLE
Kafka bindings component add version

### PR DESCRIPTION
**Pull request by light-pan**
Signed-off-by: panxiangyue <panxiangyue@corp-ci.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Issue reference

[2139](https://github.com/dapr/docs/issues/2139)

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md [hunk 1]
```diff
@@ -54,6 +56,7 @@ spec:
 | saslPassword | N | Input/Output | The SASL password used for authentication. Can be `secretKeyRef` to use a [secret reference]({{< ref component-secrets.md >}}). Only required if `authRequired` is set to `"true"`. | `""`, `"KeFg23!"` |
 | initialOffset   | N | The initial offset to use if no offset was previously committed. Should be "newest" or "oldest". Defaults to "newest". | `"oldest"` |
 | maxMessageBytes | N | Input/Output | The maximum size in bytes allowed for a single Kafka message. Defaults to 1024. | `2048` |
+| version               | N | Kafka cluster version. Defaults to 1.0.0.0 | `1.0.0`
```
Comment (mukundansundar) [inline daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md:59]: is it 1 followed by 4 zeros or 3 ? If 4 could you provide a link to sarama docs on what the version number needs to be ?

Comment (light-pan) [reply daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md:59]: The code parse kafka version in **github.com/Shopify/sarama v1.23.1**  utils.go

```go
func ParseKafkaVersion(s string) (KafkaVersion, error) {
	if len(s) < 5 {
		return MinVersion, fmt.Errorf("invalid version `%s`", s)
	}
	var major, minor, veryMinor, patch uint
	var err error
	if s[0] == '0' {
		err = scanKafkaVersion(s, `^0\.\d+\.\d+\.\d+$`, "0.%d.%d.%d", [3]*uint{&minor, &veryMinor, &patch})
	} else {
		err = scanKafkaVersion(s, `^\d+\.\d+\.\d+$`, "%d.%d.%d", [3]*uint{&major, &minor, &veryMinor})
	}
	if err != nil {
		return MinVersion, err
	}
	return newKafkaVersion(major, minor, veryMinor, patch), nil
}
```

```go
SupportedVersions = []KafkaVersion{
		V0_8_2_0,
		V0_8_2_1,
		V0_8_2_2,
		V0_9_0_0,
		V0_9_0_1,
		V0_10_0_0,
		V0_10_0_1,
		V0_10_1_0,
		V0_10_1_1,
		V0_10_2_0,
		V0_10_2_1,
		V0_11_0_0,
		V0_11_0_1,
		V0_11_0_2,
		V1_0_0_0,
		V1_1_0_0,
		V1_1_1_0,
		V2_0_0_0,
		V2_0_1_0,
		V2_1_0_0,
		V2_2_0_0,
		V2_3_0_0,
	}
```
obviously，the value in yaml is `1.0.0` `1.1.0` and so on

**Comment by greenie-msft** [comment]
@light-pan Can you please target this for the v1.7 branch/release?

**Comment by light-pan** [comment]
> 

ok

**Pull request closed without merging**